internal/website/content/ref/blob.md

```diff
@@ -1,5 +1,8 @@
 ---
 title: "Blob"
+date: 2019-02-21T16:21:29-08:00
```

Comment:
Just a question: is it necessary to specify these dates in such precision, or at all? What do they really mean for reference pages? Is this for ordering?

Reply:
Precision not required, but comes default with Hugo. It's reflected in the RSS feed and Sitemap automatically generated for the site, which currently read "Jan 1 0001", which is bad. 🙂

Reply:
So should we update this date/time whenever we update a page? Or post it initially? This would be good to document somewhere.

Reply:
See [Front Matter]. If you use `hugo new`, `date` is added for you. `lastmod` should be tweaked when pages get substantially new information.

Added to internal/website/README.md